internal/token.go

```diff
@@ -50,6 +50,11 @@ type Token struct {
 	// mechanisms for that TokenSource will not be used.
 	Expiry time.Time
 
+	// ExpiresIn is the OAuth2 wire format "expires_in" field,
+	// which specifies how many seconds later the token expires,
+	// relative to an unknown time base approximately around "now".
```

Comment:
I would make it more clear here what "now" means. Indeed it's the time when the token was fetched.